src/parser/github-comment-module.ts

```diff
@@ -164,14 +164,7 @@ export class GithubCommentModule implements Module {
             <td>
             <details>
               <summary>
-                ${Object.values(commentScore.score?.formatting?.content || {}).reduce((acc, curr) => {
-                  const multiplier = new Decimal(
-                    commentScore.score?.formatting
-                      ? commentScore.score.formatting.formattingMultiplier * commentScore.score.formatting.wordValue
-                      : 0
-                  );
-                  return acc.add(multiplier.mul(curr.score * curr.count));
-                }, new Decimal(0))}
+                ${new Decimal(commentScore.score?.reward || 0).div(new Decimal(commentScore.score?.relevance || 1))}
```

Comment:
shouldn't it multiply by relevance not divide?

Reply:
This is a shortcut to find back the Formatting value without having to recompute everything, see screenshot:
![image](https://github.com/user-attachments/assets/5def6aa0-97dd-4acf-a17e-cf61930ba83b)